<!DOCTYPE html>
<html><head><title>_clientDiagnostics</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">1 write to _clientDiagnostics</div><div class="rA">Azure.ResourceManager.Core (1)</div><div class="rG" id="Azure.ResourceManager.Core"><div class="rF"><div class="rN">Generated\Resources\SubscriptionsRestOperations.cs (1)</div>
<a href="../Generated/Resources/SubscriptionsRestOperations.cs.html#28"><b>28</b><i>_clientDiagnostics</i> = clientDiagnostics;</a>
</div>
</div>
<div class="rH">8 references to _clientDiagnostics</div><div class="rA">Azure.ResourceManager.Core (8)</div><div class="rG" id="Azure.ResourceManager.Core"><div class="rF"><div class="rN">Generated\Resources\SubscriptionsRestOperations.cs (8)</div>
<a href="../Generated/Resources/SubscriptionsRestOperations.cs.html#71"><b>71</b>throw await <i>_clientDiagnostics</i>.CreateRequestFailedExceptionAsync(message.Response).ConfigureAwait(false);</a>
<a href="../Generated/Resources/SubscriptionsRestOperations.cs.html#98"><b>98</b>throw <i>_clientDiagnostics</i>.CreateRequestFailedException(message.Response);</a>
<a href="../Generated/Resources/SubscriptionsRestOperations.cs.html#140"><b>140</b>throw await <i>_clientDiagnostics</i>.CreateRequestFailedExceptionAsync(message.Response).ConfigureAwait(false);</a>
<a href="../Generated/Resources/SubscriptionsRestOperations.cs.html#167"><b>167</b>throw <i>_clientDiagnostics</i>.CreateRequestFailedException(message.Response);</a>
<a href="../Generated/Resources/SubscriptionsRestOperations.cs.html#201"><b>201</b>throw await <i>_clientDiagnostics</i>.CreateRequestFailedExceptionAsync(message.Response).ConfigureAwait(false);</a>
<a href="../Generated/Resources/SubscriptionsRestOperations.cs.html#221"><b>221</b>throw <i>_clientDiagnostics</i>.CreateRequestFailedException(message.Response);</a>
<a href="../Generated/Resources/SubscriptionsRestOperations.cs.html#261"><b>261</b>throw await <i>_clientDiagnostics</i>.CreateRequestFailedExceptionAsync(message.Response).ConfigureAwait(false);</a>
<a href="../Generated/Resources/SubscriptionsRestOperations.cs.html#288"><b>288</b>throw <i>_clientDiagnostics</i>.CreateRequestFailedException(message.Response);</a>
</div>
</div>
</body></html>